src/JuMP.jl

```diff
@@ -436,6 +419,25 @@ function set_parameter(model::Model, name, value)
     return MOI.set(model, MOI.RawParameter(name), value)
 end
 
+"""
+    set_parameters(model::Model, pairs::Pair...)
+
+Given a list of `parameter_name => value` pairs, calls
+`set_parameters(model, parameter_name, value)` for each pair. This is a
+convenience function only. See [`set_parameter`](@ref).
```

Comment:
What does "convenience function only" means. It seems  to say there is a restriction to using it while I guess you mean to say it's "just I convenience function" (maybe I'm biased by French here)

Reply:
I reworded this, should be clearer now.